CRM/Core/Component.php

```diff
@@ -203,24 +203,6 @@ public static function xmlMenu() {
     return $files;
   }
 
-  /**
-   * @return array
-   */
-  public static function &menu() {
```

Comment:
Grepped around on `'menu'`", `"menu"`, `menu(`, `:menu`, `>menu`. Couldn't find anything relevant callers. Looks good.

Also, did a little bit `r-run` by enabling/disabling components and observing how the menu items were enabled/disabled. Seemed fine.